flow-libs/ink.js.flow

```diff
@@ -26,7 +26,16 @@ declare module 'ink' {
     cyan?: boolean,
     magenta?: boolean,
     reset?: boolean,
-    yellow?: boolean
+    yellow?: boolean,
+    blue?: boolean,
+    blackBright?: boolean,
+    redBright?: boolean,
+    greenBright?: boolean,
+    yellowBright?: boolean,
+    blueBright?: boolean,
+    magentaBright?: boolean,
+    cyanBright?: boolean,
+    whiteBright?: boolean,
```

Comment:
I wonder if we could import the `chalk` types to avoid needing to maintain this: https://github.com/flow-typed/flow-typed/blob/29009cc98df52d81d5a7d3e054d1bbf5199877d4/definitions/npm/chalk_v2.x.x/flow_v0.104.x-/chalk_v2.x.x.js#L26